packages/code/src/main.rs

```diff
@@ -94,13 +94,113 @@ fn transform_code(to: &str) {
                 print!("[{}]", json!({"name": "raw", "data": html}));
             }
         }
+        "latex" => {
+            if font_size != 12{
+                eprintln!("Font size is not supported in LaTeX");
+            }
+            print!("{}", highlight_latex(code, lang, theme, tab_size));
+        }
         other => {
             eprintln!("Cannot convert code to {other}");
         }
     }
 }
 
-fn get_style(
+fn highlight_latex(code: &str, lang: &String, tm: &str, tab_size: u64) -> String {
```

Comment:
I think the naming may be confusing, `highlight_latex` vs `get_highlighted_html`, it seems to me that they do similar things, right? They may be named in a way which is more clear that they are equivalent.

Also, `&str` is better than `&String`